src/main/java/io/reactivex/disposables/CompositeDisposable.java

```diff
@@ -86,7 +86,7 @@ public boolean isDisposed() {
 
     @Override
     public boolean add(Disposable d) {
-        ObjectHelper.requireNonNull(d, "d is null");
+        ObjectHelper.requireNonNull(d, "Disposable is null");
```

Comment:
We want to avoid that in case in some odd JVM language the parameter name change is a breaking change.